Test.md

```diff
@@ -0,0 +1 @@
+Testas
```

Comment:
Verta mokintis nepalikti šiukšlių.

Plačiau:
 * https://en.wikipedia.org/wiki/Technical_debt

Reply:
Kažkaip daug studentų kuria `Test.md` failą, nors jo ir nereikia kurti :thinking:

> Aš skiriu ir taip daug laiko, kad įspėčiau apie kiekvieno iš jūsų **individualias klaidas** – tikiuosi, kad namų darbai nedaromi tik dėl nusirašinėjimo...

Reply:
Vadovavausi šiuo video: https://www.youtube.com/watch?v=L9UACxeA4-M&feature=youtu.be

Tas pats pasikartos ir kituose namų darbuose.

Reply:
Dėkui už paaiškinimą :+1: 
Pats nebuvau pagalvojęs.